examples/async-rest/async-rest-jar/pom.xml

```diff
@@ -11,7 +11,7 @@
   <name>Example Async Rest :: Jar</name>
   <url>http://www.eclipse.org/jetty</url>
   <properties>
-    <bundle-symbolic-name>${project.groupId}.examples.asyc.rest</bundle-symbolic-name>
+    <jpms-module-name>org.eclipse.jetty.example.examples.async.rest.example.async.rest.jar</jpms-module-name>
```

Comment:
Nope. Must be:

```
<bundle-symbolic-name>${project.groupId}.examples.async.rest</bundle-symbolic-name>
```

Only fixing the typo.
The PR is about upgrading jar and surefire plugin so changing this is unnecessary.

Reply:
groupId is ` org.eclipse.jetty.example-async-rest ` so it's not valid because of `-`